po/de.po

```diff
@@ -318,53 +304,56 @@ msgstr "Heimordner für %ls"
 #: src/complete.cpp:63
 #, c-format
 msgid "Variable: %ls"
-msgstr ""
+msgstr "Variable: %ls"
 
 #: src/complete.cpp:66
 #, c-format
 msgid "Abbreviation: %ls"
-msgstr ""
+msgstr "Abkürzung: %ls"
 
 #: src/complete.cpp:1514
 msgid "completion reached maximum recursion depth, possible cycle?"
 msgstr ""
+"Vervollständigung hat die maximale Rekursionstiefe erreicht, möglicher Kreis?"
 
 #: src/env.cpp:1340
 msgid ""
 "Could not determine current working directory. Is your locale set correctly?"
 msgstr ""
+"Das aktuelle Arbeitsverzeichnis konnte nicht bestimmt werden. Ist die "
+"Lokalisierung korrekt eingestellt?"
```

Comment:
I'd leave it at "locale" - this is about *encoding* - when $LC_CTYPE is set to e.g. SHIFT-JIS and your PWD is in UTF-8.